src/lib/util/Games/TriviaManager.ts

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { fetch, FetchResultTypes } from '@utils/util';
+
+/**
+ * This class will also be used for handling complete games of Trivia
+ */
+// eslint-disable-next-line @typescript-eslint/no-extraneous-class
```

Comment:
This rule is enforced in Skyra and should not be disabled. Static classes are similar to namespaces, and as such, just using `export async function getQuestion(...) { ... }` is prefered.